src/main/java/bhs/devilbotz/commands/auto/RotateDegrees.java

```diff
@@ -0,0 +1,55 @@
+package bhs.devilbotz.commands.auto;
+
+import bhs.devilbotz.Robot;
+import bhs.devilbotz.subsystems.DriveTrain;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class RotateDegrees extends CommandBase {
+  private DriveTrain drivetrain;
+  private PIDController rotatePid;
+  private double startAngle;
+  private double targetAngle;
+
+  public RotateDegrees(DriveTrain drivetrain, int degrees) {
+    this.drivetrain = drivetrain;
+    rotatePid =
+        new PIDController(
+            Robot.getDriveTrainConstant("ROTATE_P").asDouble(),
+            Robot.getDriveTrainConstant("ROTATE_I").asDouble(),
+            Robot.getDriveTrainConstant("ROTATE_D").asDouble());
+    rotatePid.setTolerance(1);
+    startAngle = drivetrain.getYaw();
+    targetAngle = startAngle + degrees;
+    addRequirements(drivetrain);
+  }
+
+  private void addRequirements(DriveTrain drivetrain2) {}
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    System.out.println("RotateDegrees start");
+    drivetrain.arcadeDrive(0, 0);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    double turnError = rotatePid.calculate(drivetrain.getYaw(), targetAngle);
+    drivetrain.arcadeDrive(0, -turnError);
```

Comment:
Interesting we don't need any clamping of output here to prevent robot moving too fast.  I remember seeing it turning quickly on the platform.  That's some good PID tuning y'all did to not need any other adjustments to output.

Reply:
We will likely need clamping when testing this on the actual robot with the arm.  I have a feeling since the comp bot is top heavy, spinning so fast will cause it to tip over.  I hope to test/tune this tonight on the comp bot.